01-IAM/README.md

```diff
@@ -1,64 +1,70 @@
 
 # Cloud Foundation Toolkit Lab - 01-IAM
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fterraform-google-modules%2Fcloud-foundation-training&cloudshell_open_in_editor=main.tf&cloudshell_tutorial=README.md&cloudshell_working_dir=01-IAM)
 
 ## Prerequisite
 
-Complete previous labs
-* **[00-Setup](../00-Setup/README.md)**
+Complete previous labs:
 
-You should have [Google Cloud SDK](https://cloud.google.com/sdk/docs/downloads-interactive) installed and configured before continuing, otherwise refer to [00-Setup](../00-Setup/README.md)
-
-## What You'll Learn
+* [00-Setup](../00-Setup/README.md)
```

Comment:
Internal link does not display on Tutorial pane, should change to full url. Same for rest of the files.

Reply:
I'll switch to absolute links rather than relative links given the issue with them on Cloud Shell walkthroughs